package/README.md

```diff
@@ -105,7 +105,7 @@ import type { Configuration } from "@happykit/flags/config";
 export type AppFlags = { [key: string]: boolean | number | string | null };
 
 export const config: Configuration<AppFlags> = {
-  envKey: process.env.NEXT_PUBLIC_FLAGS_ENV_KEY!,
+  envKey: process.env.NEXT_PUBLIC_FLAGS_ENVIRONMENT_KEY!,
```

Comment:
I think we should go the other way and rename the .env to `NEXT_PUBLIC_FLAGS_ENV_KEY ` as `NEXT_PUBLIC_FLAGS_ENVIRONMENT_KEY ` only pops up once but `NEXT_PUBLIC_FLAGS_ENV_KEY` pops up three times.